modules/nf-core/bcftools/reheader/main.nf

```diff
@@ -40,10 +57,16 @@ process BCFTOOLS_REHEADER {
     """
 
     stub:
+    def args = task.ext.args ?: ''
     def prefix = task.ext.prefix ?: "${meta.id}"
 
+    def extension = args.contains("--output-type b") || args.contains("-Ob") ? "bcf.gz" :
+                    args.contains("--output-type u") || args.contains("-Ou") ? "bcf" :
+                    args.contains("--output-type z") || args.contains("-Oz") ? "vcf.gz" :
+                    args.contains("--output-type v") || args.contains("-Ov") ? "vcf" :
+                    "vcf.gz"
```

Comment:
```suggestion
    def extension = args2.contains("--output-type b") || args2.contains("-Ob") ? "bcf.gz" :
                    args2.contains("--output-type u") || args2.contains("-Ou") ? "bcf" :
                    args2.contains("--output-type z") || args2.contains("-Oz") ? "vcf.gz" :
                    args2.contains("--output-type v") || args2.contains("-Ov") ? "vcf" :
                    "vcf"
```

Reply:
Don't we want to default to bgzipped vcf, with the final "vf.gz"?

Reply:
Right now I see it. Ignore my comment 🙃

Reply:
Yeah the default of `bcftools view` is to emit a plain VCF :) Overall good work with the changes